ceph/metadata.csv

```diff
@@ -21,3 +21,8 @@ ceph.write_op_per_sec,gauge,,operation,second,Per-pool write operations/second,0
 ceph.num_near_full_osds,gauge,,item,,Number of nearly full osds,-1,ceph,near full osds
 ceph.num_full_osds,gauge,,item,,Number of full osds,-1,ceph,full osds
 ceph.osd.pct_used,gauge,,percent,,Percentage used of full/near full osds,-1,ceph,osd pct used
+ceph.misplaced_objects,gauge,,item,,Number of objects misplaced,-1,ceph,misplaced objects
+ceph.misplaced_objects_total,gauge,,item,,Total number of objects if there are misplaced objects,0,ceph,misplaced objects total
```

Comment:
I'm a bit confused by this metric vs `ceph.misplaced_objects`. Is the description saying that this metric contains the total number of objects, misplaced or otherwise, if there are misplaced objects?

Reply:
Yes, so `misplaced_objects / misplaced_total` matches the percent misplaced reported by ceph status, but ceph only reports the `recovery` object where these values are if there are misplaced objects to recover. I guess my wording is a little clunky, I tried to fit that into a short description.

Reply:
Whoops, I also miswrote `misplaced_objects_total`, when it should be `misplaced_total`. Not that that's a more descriptive field name, but fixing now.